doc/source/ray-contribute/getting-involved.rst

```diff
@@ -246,13 +246,67 @@ check recent tests known to be flaky.
 
 .. _`CI`: https://github.com/ray-project/ray/tree/master/ci
 
+.. _api-stability:
```

Comment:
should we make this into a separate page?

Reply:
+1

Reply:
done